js/plugins.js

```diff
@@ -553,8 +553,47 @@ plugins.factory('userPlugins', function() {
         }
     });
 
+    /*
+     * TikTok embedded player
+     * Very similar to twitter
+     */
+    var tikTokPlugin = new UrlPlugin('TikTok', function(url) {
+        var regex = /^https?:\/\/(www\.)?tiktok\.com\/(.+)/i,
+            match = url.match(regex);
+
+        if (match) {
+
+            return function() {
+
+                var element = this.getElement();
+
+                GetEmbedContent( "https://www.tiktok.com/oembed?url=" + url, element);
+
+                function GetEmbedContent(embedurl, element) {
+                    fetch(embedurl)
```

Comment:
Apparently IE11 doesn't support fetch. I guess for the three people still using that the plugin just wouldn't work? Since the rest of GB would still work for them, I guess that's fine.